README.md

```diff
@@ -37,10 +37,12 @@ python index.py
 
 ## Development
 
-If your development environment doesn't already automatically use PEP8 for formatting, you can 
-install a git hook to automatically format Python files as part of the commit process. To do so,
-run this once:
+Consistent PEP8 compliant formatting can be ensured by enabling pre-commit
+git hook.
 
     ```
-    git config core.hooksPath .githooks
+    pip install pre-commit
```

Comment:
Will this also need a `pip install black`?

Reply:
I tested this on my machine and it does not. I supply the repositories to `pre-commit` to the ` .pre-commit-config.yaml` file and it can execute and checks and edits without installing `black` or `flake8`.